kivymd/uix/card.py

```diff
@@ -624,8 +624,8 @@ class MDCard(
     BackgroundColorBehavior,
     RectangularElevationBehavior,
     FocusBehavior,
-    BoxLayout,
     RectangularRippleBehavior,
+    BoxLayout,
```

Comment:
It won't work. Items will not scroll in the list.

Reply:
can u give the example code which doesnt work for u

Reply:
Run the Kitchen demo application and you will see that the list with examples of widgets does not scroll if you use the touchbar.

Reply:
sorry for many questions bro, for what is meant by touch bar

Reply:
Sorry, I meant trackpad on laptops:

![apple-macbook-pro-13-retina-space-gray-with-touch-bar-and-touch-id-mv972-2019-18358475253561](https://user-images.githubusercontent.com/16930280/97406200-e4dcae80-1909-11eb-9613-11d7e0abe9b2.jpeg)

Reply:
in my laptop it works good,

![crt](https://user-images.githubusercontent.com/63696279/97407209-5de70080-1920-11eb-929f-359dd9ab86ac.gif)

Reply:
The changes you are proposing have already been made by me. I removed them because if we do what you suggest the treackpad will not work. At least it doesn't work on my MacBook.

Reply:
👍🏽